WebHostLib/misc.py

```diff
@@ -36,15 +36,15 @@ def start_playing():
     return render_template(f"startPlaying.html")
 
 
-@app.route('/weighted-settings')
-def weighted_settings():
-    return render_template(f"weighted-settings.html")
+@app.route('/weighted-options')
+def weighted_options():
+    return render_template(f"weighted-options.html")
 
 
-# Player settings pages
-@app.route('/games/<string:game>/player-settings')
-def player_settings(game):
-    return render_template(f"player-settings.html", game=game, theme=get_world_theme(game))
+# Player options pages
+@app.route('/games/<string:game>/player-options')
+def player_options(game):
+    return render_template(f"player-options.html", game=game, theme=get_world_theme(game))
```

Comment:
I would like to see a flask.redirect from the old url to the new one with code 301 Moved Permanently for a version or two, so link in browser history get updated. (I have not checked if the site map "breaks" doing that)